micall/utils/aln2counts_simplify.py

```diff
@@ -38,11 +42,15 @@ def __init__(self,
         super(MicallDD, self).__init__()
         self.filename = filename
         self.simple = simple
-        base, ext = os.path.splitext(simple)
-        self.best = base + '_best' + ext
         self.get_result = getattr(self, 'check_' + test_name)
         self.reads = read_aligned(self.filename)
 
+        expected_subsequence = os.environ.get(SUBSEQ_ENV_VARNAME, None)
+        if expected_subsequence is None:
+            raise RuntimeError(f"Expected ${SUBSEQ_ENV_VARNAME!r} environment variable to be set for the {'subseq'!r} test")
+
+        self.expected_subsequence_re = re.compile(expected_subsequence)
```

Comment:
Can you just use `expected_subsequence in stitched_consensus` instead of a regex search?

Reply:
Yes, but I think it does not hurt to be more general,
because every subsequence matched as regex will do the same thing as what you suggested.

But on the other hand, it also allows patters like `MAX[^\n]+ACGTACGT`
which specify not only the sequence itself, but its respective cutoff as well.

Do you think it's worth it?